controller/src/beerocks/cli/beerocks_cli_bml.h

```diff
@@ -214,6 +217,10 @@ class cli_bml : public cli {
                               const std::string &channel_pool);
     int get_dcs_scan_results(const std::string &radio_mac, uint32_t max_results_size,
                              bool is_single_scan = false);
+    int client_get_client_list();
+    int client_set_client(const std::string &sta_mac, int8_t selected_bands,
+                          bool stay_on_initial_radio, bool stay_on_selected_device);
```

Comment:
the function should receive int8_t to support "not-configured", cfr. `set_dcs_continuous_scan_params` function

Reply:
Changed the function to support "not-configured" params